api/http/handler/stacks/create_kubernetes_stack.go

```diff
@@ -71,3 +170,15 @@ func (handler *Handler) deployKubernetesStack(endpoint *portainer.Endpoint, stac
 	return handler.KubernetesDeployer.Deploy(endpoint, stackConfig, namespace)
 
 }
+
+func (handler *Handler) cloneManifestContentFromGitRepo(gitInfo *kubernetesGitDeploymentPayload, projectPath string) (string, error) {
```

Comment:
@chiptus Hi Chaim. could you please review this func to make sure it is ok, just for double checking

Reply:
looks good